security/pfSense-pkg-snort/files/usr/local/www/snort/snort_interfaces.php

```diff
@@ -30,13 +30,10 @@
 $rcdir = RCFILEPREFIX;
 $snort_starting = array();
 
-if (!is_array($config['installedpackages']['snortglobal']['rule'])) {
-	$config['installedpackages']['snortglobal']['rule'] = array();
-}
-$a_nat = &$config['installedpackages']['snortglobal']['rule'];
+$a_nat = config_get_path('installedpackages/snortglobal/rule', []);
 
 // Calculate the index of the next added Snort interface
-$id_gen = count($config['installedpackages']['snortglobal']['rule']);
+$id_gen = count(config_get_path('installedpackages/snortglobal/rule', []));
```

Comment:
```suggestion
$id_gen = count($a_nat);
```
No need to fetch it again

Reply:
Agreed. Made the suggested change.